front/src/modules/rollingStock/components/RollingStockCard/RollingStockCardDetail.tsx

```diff
@@ -133,22 +134,24 @@ export default function RollingStockCardDetail({
         <tr>
           <td className="text-primary">{t('rollingResistanceA')}</td>
           <td>
-            {Math.floor(rs.rolling_resistance?.A ? (rs.rolling_resistance.A * 10000) / 10000 : 0)}
-            <span className="small ml-1 text-muted">N</span>
+            {floor(convertUnits('N', 'kN', rs.rolling_resistance.A), 2)}
```

Comment:
Move also the floor to convertUnits, you can add a parameter to specify the number of decimals allowed
```suggestion
            {convertUnits('N', 'kN', rs.rolling_resistance.A)}
```